packages/scss/src/components/_card.scss

```diff
@@ -29,4 +32,14 @@ $cardPadding: 1.5rem;
 	&.mod-grey {
 		background: _theme("commons.background.color");
 	}
+
+	&.mod-clickable {
+		color: _color("text.default");
+		cursor: pointer;
+		text-decoration: none;
+
+		&:hover {
+			box-shadow: 0 8px 12px rgba(0, 0, 0, .15), 0 0 2px rgba(0, 0, 0, .3);
```

Comment:
We might use a mixin to have consistant box-shadow changes over the whole framework